packages/media-utils/src/components/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as MediaUpload } from './media-upload';
+export { default as MediaEdit } from './media-edit';
+export type { MediaEditProps } from './media-edit';
```

Comment:
By looking at the `media-utils` package doc, it looks like this is a package about interacting with the media endpoints (usage section).

The `MediaEdit` is more related to fields than any other API in this repo, so I'd collocated it with the `fields` package (we already export components from there).